internal/edgeapi/backend/k8s/heartbeat-updater.go

```diff
@@ -78,3 +123,25 @@ func updateDeploymentStatuses(oldWorkloads []v1alpha1.Workload, workloads []*mod
 	}
 	return edgeWorkloads
 }
+
+func updatePlaybookExecutionStatuses(oldPlaybookExecution []v1alpha1.PlaybookExec, playbookExecutions []*models.PlaybookExecutionStatus) []v1alpha1.PlaybookExec {
+	playbookExecutionMap := make(map[string]v1alpha1.PlaybookExec)
+	for _, peStatus := range oldPlaybookExecution {
+		playbookExecutionMap[peStatus.Name] = peStatus
+	}
+	for _, status := range playbookExecutions {
+		if newPlayExec, ok := playbookExecutionMap[status.Name]; ok {
+			if string(newPlayExec.PlaybookExecutionStatus.Conditions[len(newPlayExec.PlaybookExecutionStatus.Conditions)-1].Type) != status.Status {
+				newPlayExec.PlaybookExecutionStatus.Conditions[len(newPlayExec.PlaybookExecutionStatus.Conditions)-1].Status = v1.ConditionFalse
+				now := v1.Now()
```

Comment:
I'm worried that this will match at any momemtn

Reply:
Need more info here. What do you mean with "match"?
I'm creating a new Condition and I must set the `LastTransitionTime` that, in my opinion, is `Now()`. What I'm missing?